frontend/src/components/common/form/FastCurrencyInput.tsx

```diff
@@ -79,10 +83,15 @@ const CurrencyInput = ({
   });
   value = value ? value : getIn(values, field);
   const error = getIn(errors, field);
-  const touch = getIn(touched, field);
+  let touch = getIn(touched, field);
+  if (isSubmitting && suppressValidation) {
+    touch = false;
```

Comment:
I don't really get this, you are setting this touch var but then not using it anywhere as far as I can tell.

Reply:
This is just a hacky way to optionally suppress the validation checks on unchanged rows